axlearn/common/layers.py

```diff
@@ -1598,7 +1598,7 @@ class Config(BaseLayer.Config):
     @classmethod
     def default_config(cls):
         cfg = super().default_config()
-        cfg.param_partition_spec = (None, "model")
+        cfg.param_partition_spec = ("fsdp", "model")
```

Comment:
@kelvin-zou unclear if this is safe to change by default for everyone. Please review this specifically. It was needed to fit 405b on HBM memory.